packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1677,7 +1685,7 @@ export type CaptionsReceivedListener = (event: {
 }) => void;
 
 // @public
-export type ChatAdapter = ChatAdapterThreadManagement & AdapterState<ChatAdapterState> & Disposable_2 & ChatAdapterSubscribers & AttachmentUploadAdapter;
+export type ChatAdapter = ChatAdapterThreadManagement & AdapterState<ChatAdapterState> & Disposable_2 & ChatAdapterSubscribers & _AttachmentUploadAdapter;
```

Comment:
Should the adapter still be here?

Reply:
yes it needs to be there until we refactor internal logic to remove upload adapter completely after the beta release